Helm chart for kuberhealthy 0.1.1

integrii opened the pull request:
Signed-off-by: Eric Greer <eric.greer@comcast.com>

Satisfies https://github.com/Comcast/kuberhealthy/issues/26

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

Commit:
adding kuberhealthy 0.1.1

Signed-off-by: Eric Greer <eric.greer@comcast.com>

Commit:
Add helm variables section to readme

Signed-off-by: Eric Greer <eric.greer@comcast.com>

nvtkaszpir commented on stable/kuberhealthy/templates/clusterrolebinding.yaml:
```diff
@@ -0,0 +1,14 @@
+subjects:
+- kind: ServiceAccount
+  name: kuberhealthy
+  namespace: kuberhealthy
```
I suggest to fix missing newline.

nvtkaszpir commented on stable/kuberhealthy/templates/clusterrole.yaml:
```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Please try to use v1  API if possible.

integrii committed:
squashing change iterations

integrii committed:
Update deployment.yaml

Signed-off-by: Eric Greer <eric.greer@comcast.com>

Update clusterrole.yaml

Signed-off-by: Eric Greer <eric.greer@comcast.com>

Update role.yaml

Signed-off-by: Eric Greer <eric.greer@comcast.com>

Update rolebinding.yaml

Signed-off-by: Eric Greer <eric.greer@comcast.com>

fix build errors

Signed-off-by: Eric Greer <eric.greer@comcast.com>

add OWNERS file

Signed-off-by: Eric Greer <eric.greer@comcast.com>

chart changes in accordance to label recommendations from helm

Signed-off-by: Eric Greer <eric.greer@comcast.com>

translate helm variables to helper template functions

Signed-off-by: Eric Greer <eric.greer@comcast.com>

implement rbac version helper

Signed-off-by: Eric Greer <eric.greer@comcast.com>

switch to namespace defaulting to kuberhealthy, but allow user specification

Signed-off-by: Eric Greer <eric.greer@comcast.com>

readme update for new properties

Signed-off-by: Eric Greer <eric.greer@comcast.com>

fix OWNERS structure

Signed-off-by: Eric Greer <eric.greer@comcast.com>

remove image file

Signed-off-by: Eric Greer <eric.greer@comcast.com>

viglesiasce reviewed (changes requested):
Remove PNG hinary
